pkg/api/resource_types.go

```diff
@@ -12,10 +12,11 @@ import (
 
 type Resource struct {
 	Meta
-	Version    int32
-	ConsumerID string
-	Manifest   datatypes.JSONMap
-	Status     datatypes.JSONMap
+	Version            int32
+	ObservedGeneration int32
```

Comment:
what is "observed generation"?  is it the generated Kube version of the resource?

Reply:
yes, it's

Reply:
/hold

I misunderstood this observed generation, It should be the actual version of the resource on the managed cluster, I will correct this

Reply:
/unhold

using the the actual version of the resource on the managed cluster, and rename the `ObservedGeneration` to `ObservedVersion` to avoid confusion